src/scripts/exclusion_reasons_enum_updater.py

```diff
@@ -0,0 +1,77 @@
+"""Updates exclusion reasons enum: src/schema/mondo.yaml
+
+Resources:
+- Source CSV: https://docs.google.com/spreadsheets/d/18V-_j4b22LzCytxFEM3pBEl6-_ODJY06gpUwH0F6JVY/edit#gid=1644570180
+
+Instructions:
+1. Save the 'exclusion code and definitions' tab from the above GoogleSheet as `src/scripts/exclusion_reasons.csv`
+"""
+import os
+import re
+import sys
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict
+
+import pandas as pd
+import yaml
+
+
+GSHEETS_URL = 'https://docs.google.com/spreadsheets/d/18V-_j4b22LzCytxFEM3pBEl6-_ODJY06gpUwH0F6JVY/edit#gid=1644570180'
+PROJECT_DIR = Path(os.path.dirname(__file__)).parent.parent
+DEFAULTS = {
+	'input_path_exclusion_reasons': os.path.join(PROJECT_DIR, 'src', 'scripts', 'exclusion_reasons.csv'),
+	'input_path_mondo_schema': os.path.join(PROJECT_DIR, 'src', 'schema', 'mondo.yaml'),
+}
+
+
+def run(input_path_exclusion_reasons: str, input_path_mondo_schema: str):
+	"""Runs script"""
+	# Load updated exclusions
+	# - Drops all rows where all cells are empty
+	df = pd.read_csv(input_path_exclusion_reasons).dropna(axis=0, how='all').fillna('')
+	# - Format to remove whitespace, leading indentation markers, and unneeded MONDO prefix
+	df['codes'] = df['codes'].apply(lambda x: re.sub('.*MONDO:', '', x))
+
+	# Load current exclusions
+	with open(input_path_mondo_schema, 'r') as stream:
+		mondo_schema = yaml.safe_load(stream)
+	exclusion_dicts: Dict[str, Dict[str, str]] = mondo_schema['enums']['entity_type_enum']['permissible_values']
+
+	# Warning: schema enum items not located in GoogleSheet
+	missing_exclusion_reasons = [x for x in exclusion_dicts if x not in set(df['codes'])]
```

Comment:
FYI: Prints a warning if some exclusion reasons not found in GoogleSheet. I did find some. Added comment to: #5270

Reply:
Update: [Per Sabrina](https://github.com/monarch-initiative/mondo/issues/5270#issuecomment-1220087428), I have removed this warning, and it now instead deletes these:
> these missing ones are "old" codes that should be deleted (and not used anymore). Thank you for checking! I should have communicated that.